File: WindowDemo/WindowDemo/BubbleViewModel.m
//
//  BubbleViewModel.m
//  WindowDemo
//
//  Created by Theo on 2024/8/23.
//

#import "BubbleViewModel.h"


@implementation BubbleViewModel
- (instancetype)initWithModel:(BubbleModel *)model {
    self = [super init];
    if(self){
        self.model = model;
    }
    return self;
}


@end

File: BubbleView/BubbleView/BubbleViewModel.swift
//
//  BubbleViewModel.swift
//  BubbleView
//
//  Created by Theo on 2024/8/27.
//

import Foundation

class BubbleViewModel: NSObject {
    var model: BubbleModel
    
    init(model: BubbleModel) {
        self.model = model
        super.init()
    }
}
